new-dti-website/src/app/page.tsx

```diff
@@ -116,11 +116,12 @@ const Home: React.FC = () => {
           <div className="flex-grow">
             <Slideshow selectedImage={selectedIcon} />
           </div>
+          <RedBlob intensity={0.6} className="left-[1200px] top-[0px]" />
```

Comment:
This makes the page horizontally scrollable. One way that I found to solve this is to wrap the blob in a div and make the most parent div have `overflow-hidden`. This also introduces some z-index issues that show up.